### PR TITLE
#2409 Get bank account from api / Inform backend about purchase request / Replace verification flow icons

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: app/components/modals/bank-transfer-flow/purchase/BankTransferPurchaseSummary.tsx [hunk 1]
```diff
@@ -78,15 +72,15 @@ const BankTransferPurchaseLayout: React.FunctionComponent<IProps> = ({
       />
       <InfoRow
         caption={<FormattedMessage id="bank-transfer.summary.recipient" />}
-        value={<CopyToClipboardLabel label={recipient} />}
+        value={<CopyToClipboardLabel label={quintessenceBankAccount!.name} />}
```
Comment (AlexeyTuboltsev) [inline app/components/modals/bank-transfer-flow/purchase/BankTransferPurchaseSummary.tsx]: I suggest to rule out the quintessenceBankAccount === undefined in the appConnect, as a branch or as an invariant. Than you don't need a casting here

File: app/lib/api/KycApi.ts [hunk 2]
```diff
@@ -283,4 +284,17 @@ export class KycApi {
 
     return response.body;
   }
+
+  public async nEurPurchaseRequest(amount: string, purpose: string): Promise<{}> {
+    const response = await this.httpClient.post<{}>({
+      baseUrl: BASE_PATH,
```
Comment (AlexeyTuboltsev) [inline app/lib/api/KycApi.ts]: is it possible to type the response? Why is it a {} ?

File: app/modules/bank-transfer-flow/sagas.ts [hunk 3]
```diff
@@ -111,8 +115,16 @@ export function* getRedeemData({ contractsService }: TGlobalDependencies): any {
   yield put(actions.bankTransferFlow.setRedeemData(bankFeeUlps, minEuroUlps));
 }
 
+export function* completeBankTransfer({ apiKycService }: TGlobalDependencies): any {
+  const reference: string = yield select(selectBankTransferFlowReference);
+
+  // TODO: replace by correct amount when implemented
+  apiKycService.nEurPurchaseRequest("1", reference);
```
Comment (AlexeyTuboltsev) [inline app/modules/bank-transfer-flow/sagas.ts]: shouldn't it have some error handling? it's a network call

File: app/lib/api/KycApi.ts (hunk 2 above)
Comment (desfero) [reply app/lib/api/KycApi.ts]: hm, the response is `{}` exactly :D

**Pull request merged**